src/source/publishing/influx_publisher.cr

```diff
@@ -67,6 +67,12 @@ module PlaceOS::Source
         hmac_sha256(match_string)
       end
 
+      # Influx doesn't support `nil` values
+      if payload.nil?
```

Comment:
I think we need to check for `"null"` here as I assume this will be a JSON value